app/web/features/auth/notifications/NotificationSettingsListItem.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Collapse,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Typography,
+} from "@material-ui/core";
+import {
+  AccountSecurityIcon,
+  AccountSettingsIcon,
+  ChatBubbleIcon,
+  CouchFilledIcon,
+  EventIcon,
+  ExpandLessIcon,
+  ExpandMoreIcon,
+  PenIcon,
+  SinglePersonIcon,
+} from "components/Icons";
+import { AUTH } from "i18n/namespaces";
+import { useTranslation } from "next-i18next";
+import { useState } from "react";
+import makeStyles from "utils/makeStyles";
+
+import { GroupAction, NotificationType } from "./EditNotificationSettingsPage";
+import NotificationSettingsSubListItem from "./NotificationSettingsSubListItem";
+
+export interface NotificationSettingsListItemProps {
+  items: GroupAction[];
+  type: NotificationType;
+}
+
+const useStyles = makeStyles((theme) => ({
+  descriptionText: {
+    fontSize: theme.spacing(1.8),
+    color: theme.palette.text.secondary,
+  },
+  listItem: {
+    "&:hover": {
+      backgroundColor: "transparent",
+    },
+    "&:not(:first-child)": {
+      borderTop: `1px solid ${theme.palette.divider}`,
+    },
+  },
+}));
+
+const mapTypeToIcon: { [key: string]: JSX.Element } = {
+  account_security: <AccountSecurityIcon fontSize="large" color="action" />,
+  account_settings: <AccountSettingsIcon fontSize="large" color="action" />,
+  chat: <ChatBubbleIcon fontSize="large" color="action" />,
+  event: <EventIcon fontSize="large" color="action" />,
+  reference: <PenIcon fontSize="large" color="action" />,
+  friend_request: <SinglePersonIcon fontSize="large" color="action" />,
+  host_request: <CouchFilledIcon fontSize="large" color="action" />,
+};
+
+export default function NotificationSettingsListItem({
+  items,
+  type,
+}: NotificationSettingsListItemProps) {
+  const classes = useStyles();
+  const notificationType =
+    type as `auth:notification_settings.edit_preferences.list_items.${NotificationType}`;
+
+  const { t } = useTranslation([AUTH], {
+    keyPrefix: "notification_settings.edit_preferences.list_items",
+  });
+  const [isCollapseOpen, setIsCollapseOpen] = useState<boolean>(false);
+
+  const handleCollapseClick = () => {
+    setIsCollapseOpen(!isCollapseOpen);
+  };
+
+  const renderItems = () =>
+    items
+      .filter((item) => item.userEditable)
```

Comment:
This is so if a group has no userEditable items (i.e. account security), it won't show the group at all (i.e. an empty collapse).